docs/source/science/calculators/mobility.rst

```diff
@@ -0,0 +1,563 @@
+
+Electronic Structure Calculation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the *QuantumPatch method* [2]_, energy disorder, electronic couplings, and reorganization energies are calculated by self-consistently equilibrating the charge densities of a subset of molecules in their unique environments.
+The shell structure is similar to those described in *Keiser et al* [3]_.
+
+In total, 200 molecules core molecules are considered, embedded in the generated morphology.
+For these 200 molecules the following is computed:
+
+- **HOMO/LUMO** levels of the embedded molecules, are self-consistently computed to yield the energy disorder, and their interactions are used to compute the overlap integral distribution across relevant distances.
+- **Electronic couplings** for every pair of 200 molecules if their distance is below a reasonable threshold.
+
+From HOMO/LUMO distributions, the energy disorder is deduced.
+
+
+The parameters of the *QuantumPatch* embedding scheme is as follows:
+
+- **Core molecule**: Self-consistent DFT def2-SVP/B3LYP
+- **First shell**: Self-consistent DFT shell def2-SVP/BP86, radius 15 Å
+- **Second shell**: Self-consistent DFTB, radius 25 Å
+- **Third shell**: Static DFTB, radius 60 Å
+
+
+Structure Expansion
+~~~~~~~~~~~~~~~~~~~
+
+To bridge the scales from atomistic resolution to the device level, a stochastic expansion scheme *EDCM* is used to expand the thin-film morphologies,
+drawing electronic couplings and site energies from distributions analyzed in the QuantumPatch method.
+
+Charge Transport Simulation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Kinetic Monte Carlo (kMC) simulations* model charge transport in organic semiconductor thin films.
+The workflow uses the LightForge package to simulate field-dependent mobility, taking into account percolation and many-body effects [4]_.
+Zero-field mobility is extrapolated to the zero-field limit assuming Poole-Frenkel field dependence.
+
+Parameters of the kMC simulations:
+
+- **Fields**: three fields are applied: 0.02 0.03 0.04 eV/nm.
+- **Morphology** and **replicas**: for every field value, 10 independent morphologies are generated using the stochastic expansion scheme, including HOMO/LUMO/Js distributions derived from the *QuantumPatch* simulations.
+- **Temperature** is 300 K.
+- **Convergence criterion**: either the fluctuation parameter "iv_fluctuation" below 0.05, or "max_iterations" exceeds 5E6.
+- **Number of Charge Carriers**: 30.
+- **Neighbours**: 120, meaning a charge carrier can jump to the nearest 120 neighboring molecules.
+
+Output
+------
+
```

Comment:
Before the detailed output in the results dictionary, we could list all the Properties (in terms of DiaDEM Properties as available in the Properties-section of the docs) that are provided by this calculator: 

This Calculator provides the following Properties: 
 - ... (and include a link to the page in the Properties-section)

Reply:
done. Note: dipole is not yet added as the property, and no page exist for it here